views/about.erb

```diff
@@ -0,0 +1,39 @@
+<html>
+
+<body>
```

Comment:
Whenever you start a new nested tag you want to add indentation - so here this would be one additional indent and below on line 5 another additional indent. etc. etc.